src/components/calcite-accordion-item/calcite-accordion-item.tsx

```diff
@@ -49,6 +50,11 @@ export class CalciteAccordionItem {
   /** optionally pass an icon to display - accepts Calcite UI icon names  */
   @Prop({ reflect: true }) icon?: string;
 
+  /**
+   * @internal
+   */
+  @Prop({ reflect: true })
+  appearance: AccordionAppearance;
```

Comment:
Can we use a CSS class on the internal element instead of a private property?

Reply:
> Instead of adding properties that are undocumented, lets keep the CSS internal and use the CSS properties and set them on the parent item.

If we need to apply CSS classes on parent element `calcite-accordion` we might need to use `::slotted`  . Is it suggestible ?